codecov_cli/services/commit/__init__.py

```diff
@@ -19,7 +19,7 @@ def create_commit_logic(
     pr: typing.Optional[str],
     branch: typing.Optional[str],
     slug: typing.Optional[str],
-    token: str,
+    token: typing.Union[str, None],
```

Comment:
How about using Optional type instead? (and all the other places with Union)
```suggestion
    token: typing.Optional[str],
```

Reply:
sure!